tools/s2i-dpdk/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # dpdk-centos7
-FROM openshift/base-centos7
+FROM docker.io/centos:centos8
```

Comment:
I think this will not work because we are using the openshift-CI to build the image maybe we need to change the image here https://github.com/openshift/release/blob/93e244edbfa8c8938cda6b20779e46dbd3f25ce5/ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml#L2